src/index.html

```diff
@@ -2,15 +2,48 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta
-      name="viewport"
-      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
+    <meta name="viewport" content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
     >
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" href="./fonts/Avenir-Book.ttf">
+    <link rel="stylesheet" href="./fonts/Avenir-Heavy.ttf">
     <title>Search bar airbnb</title>
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+    data-qa="big"
+    class="big__form"
```

Comment:
Due to the task checklist you can create general class and move all similar styels to it,  and you can create class-modifier for your small form to change some of styles

![image](https://user-images.githubusercontent.com/80700247/230377600-4bbd053c-4ceb-43cc-b76f-805e7abfaf0c.png)

Fix everywhere